vulns/CVE-2021-25742.json

```diff
@@ -0,0 +1,41 @@
+{
+  "id": "CVE-2021-25742",
+  "modified": "2021-10-21T16:08:21Z",
+  "published": "2021-10-21T16:08:21Z",
+  "summary": "Ingress-nginx custom snippets allows retrieval of ingress-nginx serviceaccount token and secrets across all namespaces",
+  "details": "A security issue was discovered in ingress-nginx where a user that can create or update ingress objects can use the custom snippets feature to obtain all secrets in the cluster.",
+  "affected": [
+    {
+      "package": {
+        "ecosystem": "kubernetes",
+        "name": "k8s.io/ingress-nginx"
+      },
+      "severity": [
+        {
+          "type": "CVSS_V3",
+          "score": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:L/A:L"
+        }
+      ],
+      "ranges": [
+        {
+          "type": "SEMVER",
+          "events": [
+            {
+              "introduced": "0"
```

Comment:
[Security issue](https://github.com/kubernetes/kubernetes/issues/126811) says - `This issue cannot be fixed solely by upgrading ingress-nginx. It can be mitigated in the following versions:`
So all versions are affected